packages/react-components/react-accordion/src/stories/Accordion/AccordionDisabled.stories.tsx

```diff
@@ -1,5 +1,5 @@
 import * as React from 'react';
-import { AccordionItem, AccordionHeader, AccordionPanel, Accordion } from '@fluentui/react-accordion';
+import { AccordionItem, AccordionHeader, AccordionPanel, Accordion } from '@fluentui/react-components';
```

Comment:
_nit:_ Order these alphabetically

```suggestion
import { Accordion, AccordionHeader, AccordionItem, AccordionPanel } from '@fluentui/react-components';
```